doc/Architecture-overview/index.md

```diff
@@ -10,156 +10,157 @@ sort_idx: 15
 
-Multiple nodes with Operations service enabled may be set up in a Kaa cluster for the purpose of horizontal scaling.
+For the purpose of horizontal scaling, you can set up a Kaa cluster with Operations service enabled for every node.
 In this case, all instances of Operations service will function concurrently.
 In case of an Operations service outage, previously connected endpoints switch to other available Operations services automatically.
-Kaa server provides instruments for the load re-balancing at run time, thus effectively routing endpoints to the less loaded nodes in the cluster.
+Kaa server can re-balance the load at run time, thus effectively routing endpoints to the less loaded nodes in the cluster.
 
-## Bootstrap service
+### Bootstrap service
 
-Kaa Bootstrap service is responsible for distributing Operations services connection parameters to endpoints.
+Kaa Bootstrap service sends the information to the endpoints about Operations services connection parameters.
 Depending on the configured protocol stack, connection parameters may include IP address, TCP port, security credentials, etc.
 Kaa SDKs contain a pre-generated list of Bootstrap services available in the Kaa cluster that was used to generate the SDK library.
 Endpoints query Bootstrap services from this list to retrieve connection parameters for the currently available Operations services.
 Bootstrap services maintain their lists of available Operations services by coordinating with ZooKeeper.
 
-## Third-party components
+### Third-party components
 
-### Zookeeper
+#### Zookeeper
 
-[Apache ZooKeeper](https://zookeeper.apache.org/) enables highly reliable distributed coordination of Kaa cluster nodes.
-Each Kaa node continuously pushes information on connection parameters, enabled services and the corresponding services load.
-Other Kaa nodes use this information in order to get list of their siblings and communicate with them.
-Active Control service uses information about available Bootstrap services and their connection parameters during the SDK generation.
+Apache ZooKeeper enables highly reliable distributed coordination of Kaa cluster nodes.
+Each Kaa node continuously pushes information about connection parameters, enabled services and the corresponding services load.
+Other Kaa nodes use this information to get the list of their siblings and communicate with them.
+Active Control service uses the information about available Bootstrap services and their connection parameters during the SDK generation.
 
-### SQL database
+#### SQL database
 
-SQL database instance is used to store tenants, applications, endpoint groups and other metadata that does not grow with the increase in the amount of endpoints.
+SQL database instance is used to store tenants, applications, [endpoint groups]({{root_url}}Glossary/#endpoint-group) and other metadata that does not grow as the number of endpoints increases.
```

Comment:
"... that does not grow as the number of endpoints increases.", please leave previous "... that does not grow with the increase in the amount of endpoints." because it's more correct explanation.